test/withDataProviders.test.js

```diff
@@ -69,6 +69,23 @@ test('nested withDataProviders with eager prefetching - child component is rende
   expect(renderedMessage.textContent).toBe('1')
 })
 
+test('nested withDataProviders with eager prefetching - child component is rendered immediately, but ' +
+  'getData is called only once', async () => {
+  const getData = [getDataWithCount, {data: ''}, GET_DATA_DELAY]
+  const MessageContainer = messageContainer({needed: true, getData})
+  const ParentContainer = () => (<MessageContainer />)
```

Comment:
Why do you need this `ParentContainer`? Can't the `Container` decorate `MessageContainer` directly?

Reply:
Ah, you're right, I was copy-pasting this from a previous test, where there was a `show` parameter, which I didn't need here, and I didn't realize I no longer need `ParentContainer` at all...